draft-ietf-oauth-status-list.md

```diff
@@ -870,6 +870,10 @@ A Status List Token in the JWT format should follow the security considerations
 
 A Status List Token in the CWT format should follow the security considerations of {{RFC8392}}.
 
+## MAC based Deployments
+
+This specification allows for both cryptographic signatures and message authentication codes (MAC) to secure a Status List Token. Implementers are RECOMMENDED to match the mechanism used for the Status List Token with the one used for its Referenced Tokens. For (H)MAC based deployments, the security implications of a shared secret should be understood and should match the general security considerations of the deployment.
```

Comment:
I feel like we don't underline the challenges and typical cirumstances of MAC-based setup enough yet: Issuers and Verifiers likely have a trusted relationship and a possiblity to exchange symmetric keys out-of-band